src/nile/common.py

```diff
@@ -197,4 +197,6 @@ def get_contract_class(contract_name, overriding_path=None):
 def get_hash(contract_name, overriding_path=None):
     """Return the class_hash for a given contract name."""
     contract_class = get_contract_class(contract_name, overriding_path)
-    return compute_class_hash(contract_class=contract_class, hash_func=pedersen_hash)
+    return hex(
```

Comment:
I think we must pad the `class_hash`, similar to what we do with addresses. We have an `hex_address` method in utils, we can add another one like `hex_class_hash`. (If we don't pad, short class hashes because of leading 0s, could be as confusing as short addresses because of leading 0s).

Reply:
agree, i considered it but wasn't sure

Reply:
If we pad `get_hash`, we should update `declare`'s class_hash registration for consistency because it's storing the unpadded hex. Also, @ericglau will probably need to adjust one of the hash values in `declare_impl`: https://github.com/OpenZeppelin/openzeppelin-nile-upgrades/blob/main/src/nile_upgrades/declare_impl.py#L18

Reply:
i insist in killing the declarations file. it serves no purpose and it's actually annoying when your declare tx fails.

Reply:
That also resolves the inconsistency :)

Reply:
@ericnordelo how long class hashes should be? 67?

Reply:
i'm getting 67-length chars on the tests, so it's at least `0x`+65. how do 63chars fit in a `felt`, which can only fit 31bytes?

Reply:
i'm getting 69-length chars on the tests, so it's at least `0x`+67. how do 63chars fit in a `felt`, which can only fit 31bytes?

Reply:
i wonder why they pad to 64. how are addresses 64 then? that shouldn't fit a felt